company.rb

```diff
@@ -0,0 +1,139 @@
+class Company
+  @@count = 0
+  def initialize(comp_name, division)
+    @comp_name = comp_name
+    #@no_emp = no_emp
+    @division = division
+    @@count += 1
+  end
+  attr_accessor :comp_name, :division
+  def print_company_details
+    puts comp_name
+    #puts no_emp
+    puts division
+    puts @@count
+  end
+
+  def read_file
+    fileobject = File.new("hello world", "r")
```

Comment:
use snake case for object name. 
do not keep space between the filename.
you can use File.close method to close that file.